HACKING.md

```diff
@@ -210,11 +210,14 @@ If `junitTags` system property isn't defined, all tests without an explicitly de
 
 | Name | Description |
 | :---: | :---: |
-| acceptance | Marks tests executed on Travis | 
+| travis | Marks tests executed on Travis |
+| acceptance | Acceptance tests |
 | regression | Regression tests |
+| upgrade | Execute upgrade tests |
+| pr | Execute subset of tests for pull requests |
 | flaky | Test which are flaky |
 | cci_flaky | Test which are flaky only on specific QE environment |
```

Comment:
Can we easily rename this `flaky_on_cci`, or is that a lot of work? I would like to make this a bit more generic and define it in terms of some env var which identifies the CI the tests are being run on E.g. `flaky_on_foo` checks whether `CI` env var has value `foo`. This would allow other contributors to easily test on their own CI infra.

We also need to document a plan for managing flaky tests.

Reply:
Currently, `cci_flaky` tag is no longer used. So maybe we can remove it and add new generic tag as you suggested in the future, when it will be needed? It should be possible to do that, we have specific profile for each tag in `systemtest/pom.xml` and here we can specify the name by env variable and use this variable for tag in junit.